### ISSUE TITLE
Menu / Submenu with Search - Axe Colour Contrast Issue with unpopulated field

### ISSUE THREAD

**Issue by amieedwards-sage** [Sage/carbon#7814]
### Description

I am using the Menu / Submenu with Search component.
I am using the `Black` color variant 
When the input field is unpopulated and I run the axe DevTools, the report highlights that the text doesn't meet the minimum color contrast ratio thresholds.
https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=AxeChrome

When the input field is filled with a value and I re-run the axe DevTools report, it passes the color contrast ratio thresholds successfully.
This can be observed in Carbon Storybook.

### Reproduction

https://carbon.sage.com/?path=/story/menu--submenu-with-search

### JIRA ticket numbers (Sage only)

SBS-146074

### Carbon version

158.23.4

### React version

v17

### Design tokens version

4.34.0

### Relevant browsers

Chrome

### Relevant OSs

MacOS

### Confidentiality

- [x] I confirm there is no confidential or commercially sensitive information included.

**Comment by edleeks87** [comment]
FE-7617
